test/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.t.sol

```diff
@@ -1555,25 +1558,54 @@ contract FM_BC_Bancor_Redeeming_VirtualSupplyV1Test is ModuleTest {
     // OnlyOrchestrator Mutating Functions
 
     /* Test transferOrchestratorToken 
-        ├── when caller is not the Orchestrator_v1 
-        │      └── it should revert (tested in base Module tests)
-        └── when caller is the Orchestrator_v1 admin
-                ├── it should send the funds to the specified address
-                └── it should emit an event?
+        ├── given the caller is not a PaymentClient
+        │   └── when the function transferOrchestratorToken() gets called
+        │       └── then it should revert
+        ├── given the caller is a PaymentClient module
+        │   └── and the PaymentClient module is not registered in the Orchestrator
+        │       └── when the function transferOrchestratorToken() gets called
+        │           └── then it should revert
+        └── given the caller is a PaymentClient module
+            └── and the PaymentClient module is registered in the Orchestrator
+                └── then is should send the funds to the specified address
+                    └── and it should emit an event
     */
 
-    function testTransferOrchestratorToken(address to, uint amount)
-        public
-        virtual
-    {
+    function testTransferOrchestratorToken_FailsGivenCallerIsNotPaymentClient(
```

Comment:
You dont test this here, but in the module directly.
In here you would just make sure that the Modifier is in the correct place

Reply:
This probably feeds into this comment right @FHieser ?
https://inverternetwork.slack.com/archives/C04S8MKSB5G/p1719839850367339

And I am testing that the module is in place like this. I know we spook about this before but could you give me an example of how you envision this?

Reply:
Yep
The modifier that you test here is originally located in the Module directly, but you still test it here/ in the other fundingManagers (which means you test 3 times the same thing essentially)
Create a testModifier test in the Module_v1.t.sol (maybe via a new function you create in the ModuleV1Mock) and test it instead there. Then in the fundingManagers you only have to test if the modifier is in position in the first place.
Use the LM_PC_Bounties_v1.t.sol file I created as an example and search for 1. the modifier section (this will land in the Module.t.sol) and then for on  ModifierInPosition test (this will land in the fundingmanagers). 
Does that make it clearer? @Zitzak

Reply:
Yes will have a look at it, and adapt accordingly after our calls. Thank you @FHieser